Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:18
+
+WORKDIR /app
+
+COPY . .
+
+RUN npm install
+
+RUN npm run build

```

Comment:
`RUN npm install && npm run build`

Reply:
i was trying to avoid running both `npm install` and `npm run build` since they are very slow and would like to run from cache instead of having to run the command every time